frontend/src/component/feedbackNew/FeedbackList.tsx

```diff
@@ -0,0 +1,145 @@
+import useFeedbackPosted from 'hooks/api/getters/useFeedbackPosted/useFeedbackPosted';
+import { VirtualizedTable } from 'component/common/Table';
+import { DateCell } from 'component/common/Table/cells/DateCell/DateCell';
+import { useFlexLayout, useSortBy, useTable } from 'react-table';
+import { sortTypes } from 'utils/sortTypes';
+import { TextCell } from 'component/common/Table/cells/TextCell/TextCell';
+import { PageContent } from 'component/common/PageContent/PageContent';
+import { PageHeader } from 'component/common/PageHeader/PageHeader';
+import { ConditionallyRender } from 'component/common/ConditionallyRender/ConditionallyRender';
+import { Search } from 'component/common/Search/Search';
+import { useMediaQuery } from '@mui/material';
+import { SearchHighlightProvider } from 'component/common/Table/SearchHighlightContext/SearchHighlightContext';
+import { useSearch } from 'hooks/useSearch';
+import theme from 'themes/theme';
+import { useState } from 'react';
+
+export const FeedbackList = () => {
+    const { feedback } = useFeedbackPosted();
+
+    const [searchValue, setSearchValue] = useState('');
+
+    const columns = [
+        {
+            Header: 'Category',
+            accessor: 'category',
+            Cell: ({ row: { original: feedback } }: any) => (
```

Comment:
we don't need any. Check this line: https://github.com/Unleash/unleash/pull/6110/files#diff-0d5b54b44eae0d4cbafc9c31f6f311de6951fe8d46c632c954e29c1063762c8cR10